test/ci/set-env.sh

```diff
@@ -77,9 +77,9 @@ else
 fi
 
 #
-# XML Resolver
+# Apache XML Resolver
 #
-export XML_RESOLVER_JAR="${XSPEC_TEST_DEPS}/xml-resolver-${XML_RESOLVER_VERSION}/resolver.jar"
+export APACHE_XMLRESOLVER_JAR="${XSPEC_TEST_DEPS}/apache-xmlresolver-${APACHE_XMLRESOLVER_VERSION}/resolver.jar"
```

Comment:
Including "apache" in the directory name is helpful in case someone working in this repo forgets which organization produced resolver.jar, xml-resolver-1.2.jar (two hyphens; Apache), or xmlresolver-5.2.0.jar (one hyphen; XMLResolver.org).

Reply:
Yes, their product/file names are really confusing!